livre/eq/Config.cpp

```diff
@@ -60,6 +97,65 @@ class Config::Impl
         , dataFrameRange( INVALID_FRAME_RANGE )
     {}
 
+    void gatherHistogram( const Histogram& histogram, const float area, const uint32_t currentId )
```

Comment:
This one I will insist. You gather the histograms in this one. You gather into right buckets.